service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java

```diff
@@ -124,21 +122,13 @@ public void sendDocumentsToCafcass(final AdditionalApplicationsUploadedEvent eve
         }
     }
 
+    //DOUBLE CHECK NO OTHERS ARE INCLUDED IN THIS LIST
     private Set<Recipient> getRecipientsToNotifyByPost(CaseData caseData, AdditionalApplicationsBundle uploadedBundle) {
         Set<Recipient> allRecipients = new LinkedHashSet<>(sendDocumentService.getStandardRecipients(caseData));
```

Comment:
Looks like getStandardRecipients includes getRepresentativesServedByPost & getNotRepresentedRespondents ... is that perhaps all we need?